### DISCUSSION TITLE
Compiling Directus with @sentry/node in Dockerfile

### DISCUSSION THREAD

**Discussion by KasparRohtmaa** [directus/directus#19031] [Feature Requests]
### Summary

Hello!

If I'm posting to the wrong place or going against some guidelines please let me know as I'm very green.

TL;DR Trying to compose Docker image of Directus with Sentry module

### Motivation

Issues and Monitoring.

### Detailed Design

I made a Dockerfile (well a bunch of them, but with the same goal in mind) that currently has this:
```
FROM directus/directus:latest 
USER root
RUN npm install --save @sentry/node
USER node
```

I have also tried using yarn equivalent.
And I have a docker-compose.yml where I said to use said Dockerfile.
(I got Directus running and working with the .yml file before adding Dockerfile)

My point is I'm trying to add sentry module to the docker build and trying to get the build working so issues would go to Sentry and I performance monitoring etc would work. 

In a way Sentry already "reported" an issue when I ran my test .js code but I think the reason why that worked was how the code was not really that Sentry could see inside the Directus application.

The problem is that the build is always failing at the @sentry/node line with npm telling me "Cannot read properties of null (reading 'matches')
 and yarn telling me also some kind of error.

### Must Have:
- A
- B

### Should Have:
- C

### Could Have:
- D

### Won't Have:
- E

### Drawbacks

This is a question - trying to get help like other people in discussion.

### Alternatives

Non. Trying to get this specific case to work.

### Adoption Strategy

No idea

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
